### PR TITLE
basic model setup for productions

### PR DESCRIPTION
**Pull request by Kechr**

### CODE REVIEW

**Comment by nymann** [comment]
override (implement) the method in the dummy client:
```
[ERROR] /home/runner/work/desktop-client/desktop-client/src/main/java/dk/creditoro/client/networking/dummy_client/DummyClient.java:[9,8] dk.creditoro.client.networking.dummy_client.DummyClient is not abstract and does not override abstract method searchProductions(java.lang.String,java.lang.String) in dk.creditoro.client.networking.IClient
```

File: src/main/java/dk/creditoro/client/model/crud/Production.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package dk.creditoro.client.model.crud;
+
+public class Production {
+    private String identifier;
+    private String title;
+    private String[] producers;
```
Comment (nymann) [inline src/main/java/dk/creditoro/client/model/crud/Production.java]: A production can only have 1 producer.

**Review by nymann — CHANGES_REQUESTED** [review]
Looks good, see the comment about production only can have 1 producer.

File: src/main/java/dk/creditoro/client/model/crud/Production.java (hunk 1 above)
Comment (Joergenson) [reply src/main/java/dk/creditoro/client/model/crud/Production.java]: oh, so the API returns a producer object?

**Review by Joergenson — APPROVED** [review]

**Pull request merged**